src/apps/common/submodules/conferenceViewer/conferenceViewer.js

```diff
@@ -246,67 +246,72 @@ define(function(require) {
 
 		conferenceViewerFormatData: function(data, args) {
 			var self = this,
-				formattedData = {
-					backButton: args.backButton,
-					conference: {
-						id: data.conference.id,
-						name: data.conference.name,
-						duration: data.conference._read_only.duration,
-						isLocked: data.conference._read_only.is_locked,
-						participants: [],
-						moderators: []
-					}
-				};
-
-			data.participants = self.conferenceViewerFormatUsers(data.participants);
-
-			_.each(data.participants, function(participant) {
-				participant.is_moderator ? formattedData.conference.moderators.push(participant) : formattedData.conference.participants.push(participant);
-			});
-
-			formattedData.conference.disabledActions = formattedData.conference.participants.length + formattedData.conference.moderators.length === 0;
-
-			return formattedData;
+				isModerator = _.flow(
+					_.partial(_.ary(_.get, 2), _, 'is_moderator'),
+					_.partial(_.isEqual, true)
+				),
+				users = self.conferenceViewerFormatUsers(data.participants),
+				hasUsers = !_.isEmpty(users),
+				moderators = _.filter(users, isModerator),
+				participants = _.reject(users, isModerator);
+
+			return {
+				backButton: args.backButton,
+				conference: _.merge({
+					disabledActions: !hasUsers,
```

Comment:
This value does not seem to be read anywhere within this repository. Will it be used in the future?

Reply:
After do [looking into it](https://github.com/2600hz/monster-ui/commit/572e49cfc61898c5ecc81a1c95054dbfb104708f), it does seem like it being used.